app/controllers/application_controller.rb

```diff
@@ -1,5 +1,21 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.first
+  before_action :authenticate_user!
+  protect_from_forgery with: :exception
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :bio, :photo, :email, :password) }
+    devise_parameter_sanitizer.permit(:account_update) do |u|
+      u.permit(:name, :email, :bio, :photo, :password, :current_password)
```

Comment:
- [ ] It would be nice to permit  password_confirmation because the terminal shows an error message mentioning that the parameter is not permitted 👍

Reply:
could you please send a video of it ?

Reply:
- Hello @cvalencia1991 , sorry for the late reply. I see you already made changes and I assume you manged to fix the issue